src/main/java/com/microservices/saga/choreography/supervisor/repository/SagaStepDefinitionTransitionEventRepository.java

```diff
@@ -0,0 +1,16 @@
+package com.microservices.saga.choreography.supervisor.repository;
+
+import com.microservices.saga.choreography.supervisor.domain.definition.SagaStepDefinitionTransitionEvent;
+import org.springframework.data.neo4j.repository.Neo4jRepository;
+
+import java.util.List;
+
+public interface SagaStepDefinitionTransitionEventRepository extends Neo4jRepository<SagaStepDefinitionTransitionEvent, Long> {
+    SagaStepDefinitionTransitionEvent findSagaStepDefinitionBySagaNameAndEventName(String sagaName, String eventName);
+
+    SagaStepDefinitionTransitionEvent findSagaStepDefinitionByEventId(Long eventId);
```

Comment:
How can we find step definition by event id? And why do we return `SagaStepDefinitionTransitionEvent` in this case?